packages/website/src/layout/ElementWrapper.tsx

```diff
@@ -1,21 +1,32 @@
-import * as React from 'react';
-import _merge from 'lodash/merge';
-import _omit from 'lodash/omit';
-import queryString from 'query-string';
-
-import { Provider, ToastManager } from 'bumbag';
-import defaultTheme from '../theme';
-import { themeMap } from '../utils/theme';
+import React, { useState, useEffect } from 'react';
+import _isEmpty from 'lodash/isEmpty';
+import { getTheme, APPS } from '@localz/localz-bumbag-theme';
+import { Provider, ToastManager, Spinner, Container } from 'bumbag';
 
 export default function ElementWrapper(props: { element: React.ReactNode }) {
+  const [theme, setTheme] = useState({});
   const { element } = props;
-  const themeName = (queryString.parse(typeof window !== 'undefined' ? window.location.search : '') || {}).theme;
-  const targetTheme = themeMap[themeName] || {};
-  const theme = _merge(defaultTheme, _omit(targetTheme, 'SideNav', 'PageWithSidebar'));
 
+  useEffect(() => {
+    const getLocalzTheme = async () => {
+      try {
+        const localzTheme = await getTheme({ projectId: 'projectId', isDev: true, app: APPS.dashboard });
+        setTheme(localzTheme);
+      } catch (e) {
+        console.log(`Failed to get theme - ${JSON.stringify(e)}`);
```

Comment:
```suggestion
        console.error(`Failed to get theme - ${JSON.stringify(e)}`);
```
Perhaps we should use this, but looking otherwise! 🚀

Reply:
Can we maybe use `console.error(e, 'Failed to get theme')` instead? The [API](https://developer.mozilla.org/en-US/docs/Web/API/Console/error) accepts error objects.

Stringifying errors can swallow messages:
<img width="714" alt="Screen Shot 2021-02-09 at 4 06 24 pm" src="https://user-images.githubusercontent.com/1657832/107318412-d436f980-6af0-11eb-8a08-04acdaf216c9.png">

Whereas passing the error results in:
<img width="611" alt="Screen Shot 2021-02-09 at 4 07 21 pm" src="https://user-images.githubusercontent.com/1657832/107318455-edd84100-6af0-11eb-9b04-a72d4d2575c3.png">

Reply:
Nice one! @auzwang